cache_cleanup.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -x
+
+source logging.sh
+source common.sh
+source utils.sh
```

Comment:
I don't think utils.sh is used in any of these?

Reply:
Fixed.